### PR TITLE
Re-enable Deluge.xml

### PR DESCRIPTION
**Pull request by J0WI**
#15750

### CODE REVIEW

File: src/chrome/content/rules/Deluge.xml [hunk 1]
```diff
@@ -1,15 +1,17 @@
-<ruleset name="Deluge-torrent.org" default_off="refused">
+<!--
+	Mismatch:
+		- download.deluge-torrent.org
```
Comment (Bisaloo) [inline src/chrome/content/rules/Deluge.xml]: svn.deluge-torrent.org

Comment (J0WI) [reply src/chrome/content/rules/Deluge.xml]: Added, thanks!

**Comment by Bisaloo** [comment]
@J0WI, I think you missed this:

> Can you also rename the file to Deluge-torrent.org.xml please?

**Comment by J0WI** [comment]
Done

**Comment by Bisaloo** [comment]
Thank you!

**Pull request merged**